flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -237,6 +245,30 @@ public static void startAndInitializeCassandra() {
                 CREATE_TABLE_QUERY.replace(TABLE_NAME_VARIABLE, TABLE_NAME_PREFIX + "initial"));
     }
 
+    private static void raiseCassandraRequestsTimeouts() {
+        try {
+            final Path configurationPath = TEMPORARY_FOLDER.newFile().toPath();
+            CASSANDRA_CONTAINER.copyFileFromContainer(
+                    "/etc/cassandra/cassandra.yaml", configurationPath.toAbsolutePath().toString());
+            String configuration =
+                    new String(Files.readAllBytes(configurationPath), StandardCharsets.UTF_8);
+            String patchedConfiguration =
+                    configuration
+                            .replace("request_timeout_in_ms: 10000", "request_timeout_in_ms: 30000")
+                            .replace(
+                                    "read_request_timeout_in_ms: 5000",
+                                    "read_request_timeout_in_ms: 15000")
+                            .replace(
+                                    "write_request_timeout_in_ms: 2000",
+                                    "write_request_timeout_in_ms: 6000");
+            Files.write(configurationPath, patchedConfiguration.getBytes(StandardCharsets.UTF_8));
+            CASSANDRA_CONTAINER.withConfigurationOverride(
+                    configurationPath.toAbsolutePath().toString());
```

Comment:
I wonder how this works since we are writing a File to the local filesystem and then pointing the container to the file. Does the container have access to the local filesystem?

Reply:
If you look at the withConfigurationOverride implementation you'll see that it eventually copies the file into the container.

Reply:
Correct